src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/AirshipForms.cs

```diff
@@ -0,0 +1,18 @@
+﻿/*
+ Copyright Airship and Contributors
+*/
+
+using System;
+namespace UrbanAirship.NETStandard
+{
+    /// <summary>
+    /// Entry point for initializing Airship Forms controls.
+    /// </summary>
+    public class AirshipForms
```

Comment:
This is only needed on iOS. In order for the right binding to be made between the page and renderer, the assembly containing the renderer must be loaded at the time Xamarin.Forms is initialized in the platform specific version of the App. On iOS this isn't the case by default, so this class is just here with a dummy Init method to force that to happen. You have to do this in your app delegate before anything else, like below. Otherwise nothing will happen when you load the page.

```
public override bool FinishedLaunching(UIApplication app, NSDictionary options)
{
    UrbanAirship.NETStandard.AirshipForms.Init();
    global::Xamarin.Forms.Forms.Init();
     // ...
}
```
This isn't documented behavior, but rather community knowledge shared in the forums. We'll need to make this clear in the docs.